e2e/ctl_v3_alarm_test.go

```diff
@@ -15,44 +15,62 @@
 package e2e
 
 import (
+	"os"
 	"strings"
 	"testing"
+	"time"
 )
 
-func TestCtlV3Alarm(t *testing.T) { testCtl(t, alarmTest, withQuota(64*1024)) }
+func TestCtlV3Alarm(t *testing.T) {
+	// The boltdb minimum working set is six pages.
+	testCtl(t, alarmTest, withQuota(int64(48*os.Getpagesize())))
```

Comment:
I found ```quota=13*pages``` was the smallest value that would not hit the over quota for the first small put.  On my 'fast' system the complete TestCtlV3Alarm test would pass with that.  On the 'slow' system it would still hit the over quota even after the removal + defrag.  Increasing the quota would get the test to pass on the slow system.

For now I'll just set it to ```quota=13*pages```.

I think the problem with the slow sytem may be that it takes a while for the backend/DB to actually free up the space.  If so, how can we assure DB is synced from within the e2e test? 

As above, ```13 * 64 KiB = 832 KiB```.  832 KiB is over the 128 KiB command line length limit of etcdctl, so there's no way to do a single put in the e2e tests.